uberpoet/filegen.py

```diff
@@ -190,10 +221,16 @@ def gen_class(self, class_count, func_per_class_count):
     def gen_file(self, class_count, function_count, import_list=None):
         if import_list is None:
             import_list = []
-        imports = "\n".join(['#import \"{}\"'.format(i) for i in import_list])
-        class_out, class_nums = self.gen_class(class_count, 5)
+        imports = []
```

Comment:
For ObjC we want to import our own header as well as the module headers, this is why the logic is slightly split here.